backend/uclapi/dashboard/tasks.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+import keen
+
+from celery import shared_task
+
+
+@shared_task
+def test_task(param):
+    return 'The test task executed with argument "%s" ' % param
+
+
+@shared_task
+def keen_add_event_task(title, data):
+    try:
+        keen.add_event(title, data)
+    except Exception:

```

Comment:
I feel like this exception should be removed. If keen is down or there is a different reason something is failing, we should be seeing that as an exception in Sentry, so that we can action it.